okhttp-client/src/main/scala/org/http4s/client/okhttp/OkHttpBuilder.scala

```diff
@@ -89,12 +84,11 @@ sealed abstract class OkHttpBuilder[F[_]] private (
   private def run(req: Request[F]) =
     Resource.suspend(F.async[Resource[F, Response[F]]] { cb =>
       okHttpClient.newCall(toOkHttpRequest(req)).enqueue(handler(cb))
-      ()
+      F.delay(Option(F.unit))
```

Comment:
This is really an `F[Option[F[Unit]]`?

It looks `pure` instead of `delay`ed, at least.

Reply:
The easiest way to migrate old uncancelable `async` code, is to use `async_` , which has the same signature of the old async. The old code didn't shift back after async, whereas with ce3 it will, fyi.

Reply:
@rossabaker yes, a pure should work. My bad. But yes, it is an `F[Option[F[Unit]]]`